BE/src/main/java/com/codesquad/issuetracker/auth/controller/LoginController.java

```diff
@@ -0,0 +1,31 @@
+package com.codesquad.issuetracker.auth.controller;
+
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/login")
+public class LoginController {
+
+    private final String GITHUB_URL = "https://github.com/login/oauth/authorize";
+    private final String CLIENT_ID;
+    private final String CLIENT_ID_IOS;
+
+    public LoginController(Environment environment) {
+        this.CLIENT_ID = environment.getProperty("github.client.id");
+        this.CLIENT_ID_IOS = environment.getProperty("github.client.id.ios");
+    }
```

Comment:
this는 빼줘도 됩니다. 그리고 this 참조에 대해서도 공부해보는게 좋을 것 같네요.
Environment를 직접 주입받아 할당받는 방식보다는, 애노테이션을 활용하는 방법을 추천드리겠습니다.
```suggestion
    public LoginController(@Value("github.client.id") String clientId, @Value("github.client.id.ios") String iOSClientId) {
        this.CLIENT_ID = environment.getProperty("github.client.id");
        this.CLIENT_ID_IOS = environment.getProperty("github.client.id.ios");
    }
```

그리고 property의 계층구조도 생각해보시면 좋을 것 같습니다.
`github.client.id` → `github.client.id.web` 이 더 맞을 것 같습니다.